nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster-protocol/src/main/java/org/apache/nifi/cluster/protocol/jaxb/message/AdaptedDataFlow.java

```diff
@@ -23,8 +23,6 @@
     private byte[] flow;
     private byte[] snippets;
 
-    private boolean autoStartProcessors;
-
     public AdaptedDataFlow() {
     }
```

Comment:
This was something that remains from the 0.x codebase but it doesn't really make sense anymore in the Zero-Master Clustering paradigm.